dashboard-api/internal/types/user.go

```diff
@@ -7,10 +7,17 @@ import (
 
 type UserRepository interface {
 	Create(ctx context.Context, userID string) (User, error)
+	GetUserOrgIDs(ctx context.Context, userID string) ([]Org, error)
```

Comment:
`GetUserOrgs` - it doesn't just get the IDs.

Reply:
Yes this is left over from when I was just retuning ids